gluten-ut/common/src/test/scala/org/apache/gluten/utils/BackendTestSettings.scala

```diff
@@ -98,6 +98,13 @@ abstract class BackendTestSettings {
       exclusion.add(Exclude(testNames: _*))
       this
     }
+    def includeCH(testNames: String*): SuiteSettings = {
+      this
+    }
```

Comment:
Nothing happen?

Reply:
yes，it's used to indicate this case is supported by ch backend

Reply:
yes，it's used to indicate this case is supported by ch backend

Reply:
I just consider the method names are not general for backends, They are only used in CH test settings. How about this ?
```scala
def includeBackend(backendName, String, testNames: String*)
def excludeBackend(backendName, String, testNames: String*)
```